src/job-details/JobDetailsPage.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import NavBar from '../home-page/NavBar';
+import JobDetails from './JobDetails';
+
+const JobDetailsPage = () => (
+  <div>
+    <NavBar />
```

Comment:
You should find a way to add the NavBar only once. If the app has N pages this is going to be ugly.

Reply:
updated, created a layout component